servermaster/jobmanager.go

```diff
@@ -166,6 +184,7 @@ func (jm *JobManagerImplV2) Tick(ctx context.Context) error {
 
 // OnMasterRecovered implements lib.MasterImpl.OnMasterRecovered
 func (jm *JobManagerImplV2) OnMasterRecovered(ctx context.Context) error {
+	// Deprecated
```

Comment:
Can we remove this function from interface, will the failover (such as a master exits and recovers) will be executed in the `Init` function?